README.md

```diff
@@ -3,7 +3,8 @@
 This a basic webserver used to fake Network Health Check like the following endpoints:
 - `captive.apple.com`
 - `detectportal.firefox.com`
-- `www.msftncsi.com`
+- `www.msftncsi.com/ncsi.txt`
+- `www.msftncsi.com/connecttest.txt`
```

Comment:
According your screen capture, the probe host was:
- www.msftconnecttest.com
- ipv6.msftconnecttest.com

Reply:
Your're absolutely righ, apolgises.
I got mistaken as http:/www.msftncsi.com/connecttest.txt works aswell